src/configuration/locales/fr/translations.json

```diff
@@ -1,3 +1,33 @@
 {
-  "Take the survey!": "Réponder au questionnaire!"
+  "...Loading": "...Chargement",
+  "Beekeeper": "Apiculteur",
+  "Beginner": "Débutant",
+  "Billed: {{value}}": "Payant: {{value}}",
+  "Description {{description}}:": "Description: {{description}}",
+  "Experience: {{experience}}": "Expérience: {{experience}}",
+  "Expert": "Expert",
+  "FreeSpace": "Espace disponible",
+  "Garden": "Jardin",
+  "HAPPYCULTEUR © COPYRIGHT 2019. ALL RIGHTS RESERVED.": "Happyculteur © COPYRIGHT 2019. TOUS DROITS RÉSERVÉS.",
+  "Happyculteur partner!": "Partenaire Happyculteur!",
+  "Individual": "Particulier",
+  "Install": "Installer des ruches",
+  "Interest:": "Intérêt:",
+  "LEGAL NOTICE": "MENTIONS LEGALES",
+  "Learn": "Apprendre",
+  "Meadow": "Prairie",
+  "Medium": "Moyen",
+  "Public:": "Publique:",
+  "Roof": "Toit",
+  "SPECIAL THANKS": "Remerciement spéciaux",
+  "Scroll to load more": "Défiler vers le bas pour charger plus",
```

Comment:
Maybe `Défiler pour charger` to make it shorter?

Reply:
I think `Faire défiler` sounds better than `Défiler` :)